larray/core/array.py

```diff
@@ -5637,6 +5639,7 @@ def to_hdf(self, filepath, key, *args, **kwargs):
         >>> a = ndtest((2, 3))
         >>> a.to_hdf('test.h5', 'a')  # doctest: +SKIP
         """
+        key = str(_to_tick(key))
```

Comment:
there aren't ANY invalid characters in HDF array names?

Reply:
Well, using / seems to be a bad idea (not crashing but leading to weird bug) : https://stackoverflow.com/questions/30070834/in-names-in-hdf5-files-confusion

Reply:
I could be a good idea to forbid `/` and maybe `\`

Reply:
ok